src/legacy/core_plugins/kibana/common/field_formats/types/_numeral.ts

```diff
@@ -17,62 +17,68 @@
  * under the License.
  */
 
-import _ from 'lodash';
+// @ts-ignore
 import numeral from '@elastic/numeral';
+// @ts-ignore
 import numeralLanguages from '@elastic/numeral/languages';
+import { assign, has } from 'lodash';
+import { FieldFormat, TEXT_CONTEXT_TYPE } from '../../../../../../plugins/data/common/';
 
 const numeralInst = numeral();
 
-numeralLanguages.forEach(function (numeralLanguage) {
+numeralLanguages.forEach(function(numeralLanguage: Record<any, any>) {
   numeral.language(numeralLanguage.id, numeralLanguage.lang);
 });
 
-export function createNumeralFormat(FieldFormat, opts) {
-  class NumeralFormat extends FieldFormat {
+export function createNumeralFormat(BaseFieldFormat: typeof FieldFormat, opts: Record<any, any>) {
```

Comment:
Not sure that we need to pass BaseFieldFormat as argument. Let's import it like you do for other types of formatters

Reply:
Agree. It's not needed being passed. Removed.